pennylane/utils.py

```diff
@@ -353,3 +356,53 @@ def __str__(self):
                 output += "{}({}(wires={}))\n".format(return_map[op.return_type], op.name, op.wires)
 
         return output
+
+
+def inv(operation_list):
+    """Invert a list of operations or a template.
+
+    If the inversion happens inside a QNode, the operations are removed and requeued
+    in the reversed order for proper inversion.
+
+    Args:
+        operation_list (Iterable[~.Operation]): An iterable of operations
+
+    Returns:
+        List[~.Operation]: The inverted list of operations
```

Comment:
Could we add an example here as well. The aim is to slowly add in examples, so that overtime all user facing functions/classes have examples within their docstring (c.f. the NumPy documentation).